spec/features/tracks_spec.rb

```diff
@@ -42,10 +42,8 @@
 
       expected.to change { Track.count }.by(-1)
       expect(flash).to eq('Track successfully deleted.')
-      within('table#tracks') do
-        expect(page.has_content?(track.name)).to be false
-        expect(page.has_content?(track.description)).to be false
-      end
+      expect(page.has_content?(track.name)).to be false
```

Comment:
I would also like that you check that the table is not render, so that `table#tracks` is not in the page :wink:

Reply:
@Ana06 Please let me know if i did some mistake or not :smile: